frontend/src/utils/format_column_data.ts

```
@@ -20,255 +20,152 @@ export type Edge = {
-    string,
-    {
-      nodes: number;
-      group_radius: number;
-      middle: [number, number]; // x, y coordinates of the center of the group
-    }
-  > = new Map();
-  sub_node_radius: number = 40;
-  sub_node_spacing: number = this.sub_node_radius * 2 + 20;
-  top_sorted_groupings: string[] = [];
-  sub_node_gap: number = 150;
+  number_of_nodes: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)